npm-shrinkwrap.json

```diff
@@ -6530,7 +6530,8 @@
 						"ansi-regex": {
 							"version": "2.1.1",
 							"resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
-							"integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8="
+							"integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
+							"optional": true
```

Comment:
I think you're using a different npm version, we shouldn't be updating `npm-shrinkwrap.json` with this PR